api/envoy/extensions/clusters/dynamic_forward_proxy/v3/cluster.proto

```diff
@@ -54,3 +75,23 @@ message ClusterConfig {
   //
   bool allow_coalesced_connections = 3;
 }
+
+// Configuration for sub clusters. Hard code STRICT_DNS cluster type now.
+// We may also support LOGICAL_DNS, when we deprecate the dns_cache_config configuration.
```

Comment:
Realistically we are never going to deprecate it, so let's just remove this line.